tasks/bundles.js

```diff
@@ -72,10 +72,13 @@ const baseConfig = {
   optimization: {
     runtimeChunk: 'single',
     minimizer: [new TerserPlugin({
+      test: /\.m?js(\?.*)?$/i,
       sourceMap: true,
       terserOptions: {
         mangle: {
-          properties: /(^_|_$)/,
+          properties: {
```

Comment:
I actually believe I found a bug that can occur when minifying property names across different files. You have to [create a name cache and disable parallelization](https://github.com/philipwalton/blog/blob/1b68baa/tasks/javascript.js#L121-L135) to avoid it.

Could you either add that, or just remove the property mangling (since this is a demo and arguably should be as simple as possible). Either one is fine with me.

Reply:
👍 Sure, btw, I think the solution to ES5 code is to use [babel loose mode](https://babeljs.io/docs/en/babel-preset-env#loose), which doesn't use `Object.defineProperty()` and therefor the keys are regular props rather than strings. But removing it in this project is fine with me.